third_party/proto/ethermint/evm/v1/query.proto

```diff
@@ -8,7 +8,7 @@ import "ethermint/evm/v1/evm.proto";
 import "ethermint/evm/v1/tx.proto";
 import "google/protobuf/timestamp.proto";
 
-option go_package = "github.com/evmos/ethermint/x/evm/types";
+option go_package = "github.com/evmos/evmos/v18/x/evm/types";
```

Comment:
We should double check that the proto files match the evmos ones. For example

https://github.com/evmos/evmos/blob/v18.0.0/proto/ethermint/evm/v1/query.proto

there are some differences. the rpc query url for example. there is no /etheremint in them.